### ISSUE TITLE
Warning: Array to string conversion

### ISSUE THREAD

**Issue by contaoacademy** [isotope/core#2324]
- Contao 4.13.6
- PHP 8.1.1
- Isotope 2.8.5

**Reproduce:**
Start DEV-Mode and open "products"

```
ErrorException:
Warning: Array to string conversion

  at vendor/isotope/isotope-core/system/modules/isotope/drivers/DC_ProductData.php:1596
  at DC_ProductData->listView()
     (vendor/isotope/isotope-core/system/modules/isotope/drivers/DC_ProductData.php:173)
  at DC_ProductData->showAll()
     (vendor/contao/core-bundle/src/Resources/contao/classes/Backend.php:667)
  at Contao\Backend->getBackendModule('iso_products', null)
     (vendor/contao/core-bundle/src/Resources/contao/controllers/BackendMain.php:168)
  at Contao\BackendMain->run()
     (vendor/contao/core-bundle/src/Controller/BackendController.php:49)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor/symfony/http-kernel/HttpKernel.php:152)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:74)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:44)                
```

**Comment by aschempp** [comment]
do you have any hint which attribute this refers to?

**Comment by contaoacademy** [comment]
I don't know. How can I find this out?

**Comment by contaoacademy** [comment]
I think the problem is caused by the image?

![screen-20220713_fdn528Id](https://user-images.githubusercontent.com/23719181/178758112-d2506e25-9d6f-45dd-a335-b389ade877ca.jpg)

**Comment by aschempp** [comment]
fixed in 175a07f4b3e84fcfee63d48f8eda91f8475cf512
